guppy/src/graph/feature/build.rs

```diff
@@ -339,6 +324,7 @@ struct FeatureReq<'g> {
     to: &'g PackageMetadata,
     edge: PackageEdge<'g>,
     to_default_idx: Option<usize>,
+    // This will contain any build states that aren't.
```

Comment:
That aren't what? I'm assuming you meant "that aren't empty"?

Reply:
Yeah whoops.